src/applications/vaos/appointment-list/components/AppointmentsPage/TabItem.jsx

```diff
@@ -34,6 +34,7 @@ export default function TabItem({
       <NavLink
         id={`tab${id}`}
         aria-controls={isActive ? `tabpanel${id}` : null}
+        aria-label={`${id} appointments`}
```

Comment:
@1Copenut One question we had was if the label saying "express-care appointments" was ok, or if it needed to be "express care appointments".

Reply:
@jbalboni If we can make that string read "express care appointments", we'll be able to close it out in staging faster. I **think** this will work, but I know removing the hyphen will work.